server/src/main/java/org/apache/uniffle/server/ShuffleFlushManager.java

```diff
@@ -158,6 +158,7 @@ private void flushToFile(ShuffleDataFlushEvent event) {
           writeSuccess = true;
           LOG.warn("AppId {} was removed already, event {} should be dropped", event.getAppId(), event);
         } else {
+          String user = storageManager.getStorageUser(event.getAppId());
```

Comment:
Could the user be `null`?

Reply:
User can be null. But if we apply this PR into production, user will not be null.

Reply:
It will be null when shuffle was deleted.  We ever fix similar issue. You can see https://github.com/apache/incubator-uniffle/commit/55ec437aed9a465f0ad014534e17b82f501b0464 for more details.

Reply:
Remind.

Reply:
OK. I will make it empty string when user is null. like
```
String user = StringUtils.defaultString(storageManager.getStorageUser(event.getAppId()), StringUtils.EMPTY);
```

Reply:
Updated.